lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -19,22 +19,36 @@
                 '<span class="member-count"><%= membership_message %></span>' +
                 '<ul class="list-member-thumbs"></ul>'
             ),
+            profileImageTemplate: _.template(
```

Comment:
Looking at this file, it is a very strange mixture of template handling styles. Several of them are inline, but then two of them are in separate Underscore files. IMO it would be much clearer with a single Underscore template to render the whole thing, which is passed the team model and the context it needs.